src/hydra/fetchJsonLd.js

```diff
@@ -5,7 +5,7 @@
  * @param {object} options
  * @return {Promise.<object>} An object with a response key (the original HTTP response) and an optional body key (the parsed JSON-LD body)
  */
-export default (url, options = {})  => {
+export default function(url, options = {}) {
```

Comment:
Added a name also, it's better to avoid anonymous functions which can be a pain to debug.